test/Serilog.Tests/Parameters/PropertyValueConverterTests.cs

```diff
@@ -253,6 +255,55 @@ public void CSharpAnonymousTypesAreRecognizedWhenDestructuring()
             var structuredValue = (StructureValue)result;
             Assert.Equal(null, structuredValue.TypeTag);
         }
+
+        [Fact]
+        public void ValueTuplesAreRecognizedWhenDestructuring()
+        {
+            var o = (1, "A", new[] { "B" });
+            var result = _converter.CreatePropertyValue(o);
+
+            var sequenceValue = Assert.IsType<SequenceValue>(result);
+
+            Assert.Equal(3, sequenceValue.Elements.Count);
+            Assert.Equal(new ScalarValue(1), sequenceValue.Elements[0]);
+            Assert.Equal(new ScalarValue("A"), sequenceValue.Elements[1]);
+            var nested = Assert.IsType<SequenceValue>(sequenceValue.Elements[2]);
+            Assert.Equal(1, nested.Elements.Count);
+            Assert.Equal(new ScalarValue("B"), nested.Elements[0]);
+        }
+
+        [Fact]
+        public void AllTupleLengthsUpToSevenAreSupported()
+        {
+            var tuples = new object[]
+            {
+                ValueTuple.Create(1),
+                (1, 2),
+                (1, 2, 3),
+                (1, 2, 3, 4),
+                (1, 2, 3, 4, 5),
+                (1, 2, 3, 4, 5, 6),
+                (1, 2, 3, 4, 5, 6, 7)
+            };
+
+            foreach (var t in tuples)
+                Assert.IsType<SequenceValue>(_converter.CreatePropertyValue(t));
+        }
+
+        [Fact]
+        public void EightPlusValueTupleElementsAreIgnored()
```

Comment:
Nit `EightPlusValueTupleElementsAreIgnored` -> `EightPlusValueTupleElementsAreIgnoredWhenDestructuring`

Why the arbitrary 8?